src/args.rs

```diff
@@ -22,6 +22,12 @@ pub fn parse() -> ArgMatches<'static> {
              .long("port")
              .default_value("5000"))
 
+        .arg(Arg::with_name("quiet")
+             .short("q")
+             .long("quiet")
+             .required(false)
+             .takes_value(false))
```

Comment:
required and takes_value default to false, so drop these

Reply:
I've amended the commit to remove the requested lines.